### ISSUE TITLE
With ios Ver15.0, the app will be forcibly terminated when rewarded video ad rewards are obtained.

### ISSUE THREAD

**Issue by moguba** [admob-plus/admob-plus#437]
I tested it on ios.ios is 15.0.
In Ver 1.24.0, there was no problem, but in Ver 1.25.0, the application is forcibly terminated under certain conditions.
The problem area is the rewarded video ad part.
If you close the video ad before you get paid, you will be able to successfully return to the first game screen.
If you get paid and close the video ad, the app will crash.
If you restart the game after the forced termination, you can start the game screen in the state after getting the reward.
In other words, it is an error that the app is forcibly terminated only when returning to the top page of the app after getting the reward for the rewarded video ad.
android is fine.

**Comment by ratson** [comment]
@moguba I tried with ios 15 without issue. Would you capture the crash log for me to check?

**Comment by moguba** [comment]
At the time of ios 15.0, Ver 1.24.0 works without any problem with the following script.
In Ver 1.25.0, if you cannot get the reward (dismiss), you can normally perform the processing below alert ('child page') and jump to the top page.
If you get a reward, the app will stop working with a black screen before alert ('child page') at (reward).
In this case, the app will remain stopped.
If you remove the alert, the app will be forcibly terminated. After that, when you start it, the top page will be the screen where you got the reward.
There seems to be a problem when rewarding.

-------------------------------------------------------------------------
document.addEventListener('admob.rewarded.reward', function() {
if(nostat==1){
               window.localStorage.setItem("data4", '1')
             }
             backf()
    })
document.addEventListener('admob.rewarded.dismiss', function(){
                  alert('Look at the ad')
                  backf()
        })
document.addEventListener('admob.rewarded.impression', function(){
             nostat=1

    })

function backf(){
  alert('child page')
  var skinno=window.localStorage.getItem("data5");
  alert(skinno)
  var linkhtml="";
  if(skinno>0){linkhtml="index"+skinno+".html"}else{linkhtml="index.html"}
  alert('top page jump')
  location.href = linkhtml;
}

---------------------------------------------------------------------------------
